source/_integrations/smappee.markdown

```diff
@@ -20,20 +20,47 @@ ha_platforms:
   - switch
 ---
 
-The Smappee integration will allow users to integrate their Smappee monitors, plugs and switches into Home Assistant using the [official API](https://smappee.atlassian.net/wiki/spaces/DEVAPI/overview) or the limited local API.
+The Smappee integration will allow users to integrate their Smappee monitors, Smappee Comfort Plugs and Smappee Switches into Home Assistant using the local API, local MQTT broker or the [official cloud API](https://smappee.atlassian.net/wiki/spaces/DEVAPI/overview).
```

Comment:
MQTT is mentioned here once... How would that work?

Reply:
The Smappee Genius monitors have an MQTT broker that can be reached from the local network.

Reply:
I get that, but that is not part of this integration. It is only the only mention of it.
I suggest removing it from this text, as it might cause confusion and questions. For example; "Do I need to configure MQTT with Home Assistant?", "Do I need to configure the device with MQTT?", "I cannot find the instructions for MQTT on this page?"

Reply:
I see. The MQTT part has been removed.